authorization/jsonld-contexts/authorization.jsonld

```diff
@@ -11,6 +11,7 @@
 		"rCanAdmin": "https://ontology.eglobalmark.com/authorization#rCanAdmin",
 		"Group": "https://ontology.eglobalmark.com/authorization#Group",
 		"User": "https://ontology.eglobalmark.com/authorization#User",
-		"Client": "https://ontology.eglobalmark.com/authorization#Client"
+		"Client": "https://ontology.eglobalmark.com/authorization#Client",
+		"specificAccessPolicy": "https://ontology.eglobalmark.com/egm#specificAccessPolicy"
```

Comment:
please also create an issue to not forget to remove it from the cross-context

Reply:
done here https://github.com/easy-global-market/ngsild-api-data-models/issues/67